static/panes/editor.ts

```diff
@@ -548,23 +544,25 @@ export class Editor extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Edit
         }
 
         const usableLanguages = Object.values(languages).filter(language => {
-            return this.hub.compilerService.compilersByLang[language?.id ?? ''];
+            return this.hub.compilerService.getCompilersForLang(language.id);
         });
 
         this.languageInfoButton = this.domRoot.find('.language-info');
         this.languageInfoButton.popover({});
         this.languageBtn = this.domRoot.find('.change-language');
-        this.selectize = new TomSelect(this.languageBtn as any, {
+        const changeLanguageButton = this.languageBtn[0];
+        assert(changeLanguageButton instanceof HTMLSelectElement);
+        this.selectize = new TomSelect(changeLanguageButton, {
             sortField: 'name',
             valueField: 'id',
             labelField: 'name',
             searchField: ['name'],
             placeholder: '🔍 Select a language...',
-            options: _.map(usableLanguages, _.identity) as any[],
+            options: [...usableLanguages],
```

Comment:
oh thank you fod fixing up this cruft... presumably this was to copy the array

Reply:
Indeed! Trying to remove underscore js uses as I see them, it's so nice we can do most things without it with nicer syntax in modern js :)